x-pack/legacy/plugins/triggers_actions_ui/np_ready/public/application/sections/actions_connectors_list/components/actions_connectors_list.tsx

```diff
@@ -0,0 +1,386 @@
+            actionTypesIndex,
+            reloadConnectors: loadActions,
+          }}
+        >
+          {data.length > 0 ? (
+            <EuiInMemoryTable
+              loading={isLoadingActions || isLoadingActionTypes || isDeletingActions}
+              items={data}
+              sorting={true}
+              itemId="id"
+              columns={actionsTableColumns}
+              rowProps={() => ({
+                'data-test-subj': 'connectors-row',
+              })}
+              cellProps={() => ({
+                'data-test-subj': 'cell',
+              })}
+              data-test-subj="actionsTable"
+              pagination={true}
+              selection={
+                canDelete
+                  ? {
+                      onSelectionChange(updatedSelectedItemsList: ActionConnectorTableItem[]) {
+                        setSelectedItems(updatedSelectedItemsList);
+                      },
+                    }
+                  : null
+              }
+              search={{
+                filters: [
+                  {
+                    type: 'field_value_selection',
+                    field: 'actionTypeId',
+                    name: i18n.translate(
+                      'xpack.triggersActionsUI.sections.actionsConnectorsList.filters.actionTypeIdName',
+                      { defaultMessage: 'Type' }
+                    ),
+                    multiSelect: 'or',
+                    options: actionTypesList,
+                  },
+                ],
+                toolsLeft:
+                  selectedItems.length === 0 || !canDelete
+                    ? []
+                    : [
+                        <EuiButton
+                          key="delete"
+                          iconType="trash"
+                          color="danger"
+                          data-test-subj="bulkDelete"
+                          onClick={deleteSelectedItems}
+                          title={
+                            canDelete
+                              ? undefined
+                              : i18n.translate(
+                                  'xpack.triggersActionsUI.sections.actionsConnectorsList.buttons.deleteDisabledTitle',
+                                  { defaultMessage: 'Unable to delete actions' }
+                                )
+                          }
+                        >
+                          <FormattedMessage
+                            id="xpack.triggersActionsUI.sections.actionsConnectorsList.buttons.deleteLabel"
+                            defaultMessage="Delete ({count})"
+                            values={{
+                              count: selectedItems.length,
+                            }}
+                          />
+                        </EuiButton>,
+                      ],
+                toolsRight: [
+                  <EuiButton
+                    data-test-subj="createActionButton"
+                    key="create-action"
```

Comment:
Button should only be visible when there are connectors registered (UI & backend) and enabled on the backend.